buildSrc/version.properties

```diff
@@ -4,7 +4,7 @@ lucene            = 7.2.0
 # optional dependencies
 spatial4j         = 0.6
 jts               = 1.13
-jackson           = 2.8.10
+jackson           = 2.9.3
```

Comment:
Unfortunately ES can't be upgraded to the latest Jackson 2.9.x release: https://github.com/elastic/elasticsearch/pull/27032#issuecomment-340783545

But I think ASN support will still work with Jackson 2.8.10, so lets undo the Jackson upgrade here.